chainer/backends/cuda.py

```diff
@@ -219,7 +219,7 @@ def get_device(*args):
 
 def _get_device(*args):
     for arg in args:
-        if type(arg) in _integer_types:
+        if isinstance(arg, _integer_types):
```

Comment:
In this way `bool` is evaluated as `True`.
How about excluding `bool` explicitly?

Reply:
Good catch, done.